src/authenticator/formAuthenticator.js

```diff
@@ -60,16 +60,16 @@ FormAuthenticator.prototype.get = function(url){
   this._wait(function () {
     // if auth is in frame => switch inside
     if (that.frameSelector) {
-      that._getFields(that.frameSelector).then(that._checkDisplayed).then(function (isDisplayed) {
+      that._isPresent(that.frameSelector).then(function (isDisplayed) {
         if (isDisplayed && !switchedToFrame) {
-          browser.driver.switchTo().frame(that._getFields(that.frameSelector)).then(function () {
+          browser.driver.switchTo().frame(that._getField(that.frameSelector)).then(function () {
             switchedToFrame = true;
           });
         }
       });
     }
-    return that._getFields(that.userFieldSelector).then(that._checkDisplayed);
-  }, this.idpSelector ? 'redirect to custom IDP login page' : 'redirect to login page');
+    return that._isPresent(that.userFieldSelector);
+  }, 'Waiting to redirect to login page');
```

Comment:
why not have the detailed message to differentiate the 2 pages

Reply:
because it is always the login page, irrespectively if this "standard" or "custom" login page